rllib/connectors/agent/view_requirement.py

```diff
@@ -108,6 +108,7 @@ def transform(self, ac_data: AgentConnectorDataType) -> AgentConnectorDataType:
                 agent_index=agent_index,
                 env_id=env_id,
                 init_obs=d[SampleBatch.NEXT_OBS],
+                init_infos=d.get(SampleBatch.INFOS),
```

Comment:
`SampleCollector.add_init_obs()` now also takes the init info dict.

Reply:
+1